main.tf

```
@@ -30,6 +28,10 @@ module "cluster" {
   subnet_cidr                      = var.subnet_cidr
   subnet_name                      = var.subnet_name
   vnet_cidr                        = var.vnet_cidr
+  
+  registry_resource_group_name         = var.registry_resource_group_name
+  use_existing_acr_name                = var.use_existing_acr_name
+  use_existing_acr_resource_group_name = var.use_existing_acr_resource_group_name
```

```suggestion
  registry_resource_group_name         = var.registry_resource_group_name
  use_existing_acr_name                = var.use_existing_acr_name
  use_existing_acr_resource_group_name = var.use_existing_acr_resource_group_name
```
Also, all "-" should be with the same indent.